src/ResourceManagement/AppService/WebAppDeleteOption.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Management.AppService.Fluent
+{
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+
+    /// <summary>
+    /// Defines options for WebApp deletion.
+    /// </summary>
+    public class WebAppDeleteOption : IBeta
```

Comment:
Should be app service delete option

Reply:
Second though, should we remove this class to keep the interface consistent and simple to use. 

Provide default on the interface level.

Reply:
OK. Interface updated to use bool? .